site/DataPageContent.tsx

```diff
@@ -237,6 +237,16 @@ export const DataPageContent = ({
                                     <div className="related-data-item__type">
                                         {datapageJson.relatedData[0].type}
                                     </div>
+                                    {datapageJson.relatedData[0].imageUrl && (
+                                        <img
+                                            src={
+                                                datapageJson.relatedData[0]
+                                                    .imageUrl
```

Comment:
Instead of a single URL, can we integrate this with the GDrive image system?

Reply:
Okay! I don't have a sense of which features are MVP versus "this is what we're testing" for the prototype. 

Working out a way to make sure we can reference, track, and bake images through the datapage workflow seems like it'll be important eventually, but I trust you've got the better sense of what's the priorities are at the moment 👍